docs/DEVELOPING.md

```diff
@@ -1,5 +1,187 @@
 # Connectors Developer's Guide
 
+## Network Drives Connector
+
+Elastic Network Drives connector is provided in Elastic Connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
```

Comment:
Please use Level 3 subheading for "Availability and prerequisites" (for both Network Drive and S3) similar to "Setup and basic usage"